### ISSUE TITLE
Report() not handling errors terms in ANOVAs well

### ISSUE THREAD

**Issue by Sheepgirl87** [easystats/report#120]
Hey 

I'm getting NAs mind sentence in my report output for a repeated measures ANOVA, not sure its putting the correct ETA squared in for them either, for some Date, the numbers check out but not for ID.

**ANOVA output**
![image (1)](https://user-images.githubusercontent.com/43781055/104920325-8581be00-598f-11eb-8d06-519275764138.png)

**Report() output**

![image (3)](https://user-images.githubusercontent.com/43781055/104920294-7b5fbf80-598f-11eb-8c26-f8eedf156391.png)

Also, the may be a preference thing but i'm not sure we need scientific notation for three decimal places?

Thanks

Destiny

**Comment by mattansb** [comment]
Hi @Sheepgirl87,

Something seems off also in the varous dfs.

What do `parameters::model_parameters(model)` and `effectsize::eta_squared(model)` return?

**Comment by Sheepgirl87** [comment]
hey, will run now.... also

i just sent this on to Dom


Destiny Bradley <dezibradley87@gmail.com>
15:02 (4 minutes ago)
to Dominique
Hey Dom

Thanks for coming back to me.

I hadn't realised you could put in a ticket, that's even better ! Awesome
work!

I took a look at the numbers... I prefer a 95% CI, so thought I might
generate these myself on the effects for my paper, but thought why not
sense check whilst I make my mind up (whether to stick to the report() 90%
CI).

So I added manually calculated figures to the existing table and used
report() output to match.

I'm intrigued if the partial ETA is the same as the calculation for
standard ETA squared? only the effect of interaction was out by 0.01, so
these match back great!  However struggling with matching the upper CI
figures on separate effects and both lower and upper for the interaction
(like with the ETA its a little out report() vs Manual effort). Anything
green as below differs but not by huge amounts. Which depending on the
perhaps the confidence coefficient, margin of error or averages values this
easily done (matching manual vs programmed figs) and they are seemingly in
line, I just really like sense checking, so just wondered if you could
double check my logic?

ETA squared = SS/SS total
Standard Deviation = SQRT(SS)
For CI:
Confidence coefficient = NORM.S.INV(0.9) - also tried with little
difference '=T.INV(0.90,n-1)
Margin Error = (Confidence coefficient * standard dev) / (SQRT n)
upper bound CI = margin of error + average
lower bound CI = margin of error - average

[image: image.png]

[image: image.png]

Thanks

**Comment by Sheepgirl87** [comment]
hey

here is the output as requested:

[image: image.png]

[image: image.png]

Just looking at this output, my manual figures (as sent to Dom) could be
down to rounding... come to think of it.

Let me know what you think?

Thanks

Dezi

**Comment by mattansb** [comment]
> I'm intrigued if the partial ETA is the same as the calculation for
standard ETA squared?

eta<sup>2</sup> = SS / SStot
eta<sup>2</sup><sub>partial</sub> = SS / (SS + SSerror)
See: https://easystats.github.io/effectsize/articles/anovaES.html

> ETA squared = SS/SS total
Standard Deviation = SQRT(SS)
For CI:
Confidence coefficient = NORM.S.INV(0.9) - also tried with little
difference '=T.INV(0.90,n-1)
Margin Error = (Confidence coefficient * standard dev) / (SQRT n)
upper bound CI = margin of error + average
lower bound CI = margin of error - average

To get the CI for eta<sup>2</sup> we use the NCP method. The `estimate +- SE * Z` method used for the mean does not apply here.

**Comment by Sheepgirl87** [comment]
Awesome, that explains that !

Thanks, so on the report() output, any ideas why its being a bit cheeky?

**Comment by DominiqueMakowski** [comment]
> here is the output as requested:
[image: image.png]
[image: image.png]

Unfortunately @Sheepgirl87 we don't see the figures, could you reattach them so that we can see if the issue is related to `parameters::model_parameters`, `effectsize::eta_squared(model)` or report.

**Comment by Sheepgirl87** [comment]
Hey sorry for the delay, the report output df  you were concerned and I was
confused/concerned by the NAs....

[image: image.png]

[image: image.png]

**Comment by mattansb** [comment]
Still not seeing the images.....

**Comment by Sheepgirl87** [comment]
I am attaching them, this is a pain . let me try again ....

So i have included the output from report (), and effects and then also how
i have written the output and tabulated the results following the report
theme()

**Comment by DominiqueMakowski** [comment]
It seems like you're trying to comment/reply by email, maybe you can try going directly on the website (https://github.com/easystats/report/issues/120) and attaching the images here (rather than from an email)

**Comment by Sheepgirl87** [comment]
sorry, so just want to confirm the DFs are ok as Mattan was concerned and im trying to get write this up without the NAs so just sent your output re my revised output, as below:

![image (1)](https://user-images.githubusercontent.com/43781055/105367894-d9dba680-5bf8-11eb-9aaf-91902065bba4.png)
![image](https://user-images.githubusercontent.com/43781055/105367919-e06a1e00-5bf8-11eb-8d16-924fe151d2d6.png)

My versions:

<img width="320" alt="DB Output" src="https://user-images.githubusercontent.com/43781055/105368074-0e4f6280-5bf9-11eb-98dc-3e4f766ad629.PNG">
<img width="503" alt="DB version table" src="https://user-images.githubusercontent.com/43781055/105368092-13141680-5bf9-11eb-85cd-3cc5a2075106.PNG">

**Comment by DominiqueMakowski** [comment]
Right, so it seems like this indeed an issue in report, as the effectsize and parameters from where it pulls the data looks alright. I'll look into that asap. In the meantime, you can just "fill" up with the output from the tables ☺️

**Comment by Sheepgirl87** [comment]
Hey thanks, do you agree with my output version, based on report and effects output? :) thanks so much for looking in to this, hopefully it will help with others also

**Comment by humanfactors** [comment]
This is identical cause to #167 and #148, and I've provided account of bug in #167 so let's consolidate.
